lib/nelmon/cli/check_admin_up_oper_down.py

```diff
@@ -24,6 +25,17 @@ def main():
                                   help='Return Warning if interfaces are down')
     argparser.parser.add_argument('-c', action='store_true',
                                   help='Return Critical if interfaces are down')
+    argparser.parser.add_argument('-d', '--descr', dest='ifdescr_arg',  default=None, const=None,
+                                  help='Search over Interface descr with regex'
```

Comment:
Some of these descriptions get quite long with strange linebreaks. Try using `nm_check_admin_up_oper_down -h`. It should be enough to add a few `\n`